gkeep2notion.py

```diff
@@ -264,10 +264,21 @@ def parseBlock(p: str) -> dict:
 
 def parseTextToPage(text: str, page: Page):
     lines = text.splitlines()
+    lines.insert(len(lines), '')
+    last_block = None
     print(f"Parsing {len(lines)} blocks")
-    for p in lines:
+    for x in range(0, len(lines)):
+        p = lines[x]
         block = parseBlock(p)
-        page.add_text(block['text'], block['type'])
+        if last_block:
+            if last_block['type'] == 'text' and last_block['type'] == block['type'] and len(last_block['text']) + len(
```

Comment:
```suggestion
            if last_block['type'] == BlockType.Paragraph and last_block['type'] == block['type'] and len(last_block['text']) + len(
```
as `text` is not a correct Notion block type.